### PR TITLE
Add covid content changes (ENG/WLS)

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Adds new interstitial pages and updated content related to the coronavirus for the England/Wales household and individual schemas.

### How to review

Check the changes match the spreadsheet attached to the trello card.

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/covid-changes/schemas/en/census_household_gb_eng.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/covid-changes/schemas/en/census_individual_gb_eng.json)

### CODE REVIEW

**Comment by berroar** [comment]
[Covid changes Eng/Wls HH/HI (M)](https://trello.com/c/IJ5h131m/4212-covid-changes-eng-wls-hh-hi)

File: source/jsonnet/england-wales/individual/blocks/employment/employment_status_last_seven_days.jsonnet [hunk 1]
```diff
@@ -73,11 +79,11 @@ local proxyAnswerDescription = 'Freelance means that they are self-employed and
   page_title: 'Employment status in the last seven days',
   question_variants: [
     {
-      question: question(nonProxyTitle, nonProxyAnswerDescription),
+      question: question(nonProxyTitle, nonProxyQuestionDescription, nonProxyAnswerDescription),
       when: [rules.isNotProxy],
     },
     {
-      question: question(proxyTitle, proxyAnswerDescription),
+      question: question(proxyTitle, proxyQuestionDescription, proxyAnswerDescription),
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/individual/blocks/employment/employment_status_last_seven_days.jsonnet:86]: If we're making changes we should adhere to the style guide and use a single variable for proxy (isProxy) and functions for the things that vary.

File: source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then [
+    'Answer for the longest part, <em>by distance</em>, of their usual journey to work',
+    'If the <strong>coronavirus</strong> pandemic has affected how they travel to work, select the answer that best describes their <strong>current circumstances</strong>.',
+  ] else [
+    'Answer for the longest part, <em>by distance</em>, of your usual journey to work',
+    'If the <strong>coronavirus</strong> pandemic has affected how you travel to work, select the answer that best describes your <strong>current circumstances</strong>.',
+  ]
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet:21]: `usual journey to work` needs to change to `current journey to work`

File: source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet [hunk 3]
```diff
@@ -43,15 +41,21 @@ local question(title, description, additionalAnswerOptions=[]) = {
...
   text: 'One year ago, what was <em>{person_name_possessive}</em> usual address?',
   placeholders: [
     placeholders.personNamePossessive,
   ],
 };
-local proxyDescription = 'If they had no usual address one year ago, select the address where they were staying';
+local proxyDescription = [
+  'If they had no usual address one year ago, select the address where they were staying',
+  'If the <strong>coronavirus</strong> pandemic affected your usual address one year ago, select where you were living <strong>before your circumstances changed</strong>.',
+];
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet:58]: This should also be updated to use `isProxy` as the others have.

File: source/jsonnet/england-wales/individual/blocks/employment/workplace_type.jsonnet [hunk 4]
```diff
@@ -1,16 +1,66 @@
...
+    { description: 'For example, if they work from home three days a week and go to another place of work two days, select  “At or from home”.' },
+  ]
+  else [
+    { description: 'This is where you work most of the time in your main job.' },
+    { description: 'For example, if you work from home three days a week and go to another place of work two days, select “At or from home”.' },
+  ]
+);
+
+
+local guidanceContent(isProxy) = (
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/individual/blocks/employment/workplace_type.jsonnet:33]: It would be clearer if this was named `answerGuidance`

**Review by MebinAbraham — APPROVED** [review]

File: source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet [hunk 5]
```diff
@@ -3,13 +3,31 @@ local rules = import 'rules.libsonnet';
...
+  }
+  else 'One year ago, what was your usual address?'
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then [
+    'If they had no usual address one year ago, select the address where they were staying',
+    'If the <strong>coronavirus</strong> pandemic affected their usual address one year ago, select where you were living <strong>before your circumstances changed</strong>.',
+  ] else [
+    'If you had no usual address one year ago, select the address where you were staying',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet:21]: I can see these were wrong prior to the changes, but these should end with a full stop.
```suggestion
    'If they had no usual address one year ago, select the address where they were staying.',
    'If the <strong>coronavirus</strong> pandemic affected their usual address one year ago, select where you were living <strong>before your circumstances changed</strong>.',
  ] else [
    'If you had no usual address one year ago, select the address where you were staying.',
```

File: source/jsonnet/england-wales/individual/blocks/employment/hours_worked.jsonnet [hunk 6]
```diff
@@ -1,8 +1,23 @@
...
+    placeholders: [
+      placeholders.personNamePossessive,
+    ],
+  }
+  else 'In your main job, how many hours a week do you usually work?'
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then 'If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>'
+  else 'If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>'
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/individual/blocks/employment/hours_worked.jsonnet:16]: I think there should be a full stop at the end of these sentences, but it's not in the spreadsheet - the spreadsheet is inconsistent though e.g. the travel to work content does end in a full stop.

File: source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet [hunk 7]
```diff
@@ -11,7 +11,13 @@ local rules = import 'rules.libsonnet';
...
         contents: [
           {
-            description: 'The next set of questions is about your main job. Your main job is the job in which you usually work the most hours.',
+            description: 'The next set of questions is about your main job. Your main job is the job in which you work the most hours.',
+          },
+          {
+            description: '<strong>Coronavirus (COVID-19)</strong>',
+          },
+          {
+            description: 'Your circumstances may have changed during the coronavirus pandemic. Answer all questions based on your situation as it is now.',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet:20]: This first description should be a heading, and the title and heading should be in the same content block:
```suggestion
          {
            title: 'Coronavirus (COVID-19)',
            description: 'Your circumstances may have changed during the coronavirus pandemic. Answer all questions based on your situation as it is now.',
```

File: source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet [hunk 8]
```diff
@@ -23,10 +29,16 @@ local rules = import 'rules.libsonnet';
...
+              text: 'The next set of questions is about <em>{person_name_possessive}</em> main job. Their main job is the job in which they work the most hours.',
               placeholders: [placeholders.personNamePossessive],
             },
           },
+          {
+            description: '<strong>Coronavirus (COVID-19)</strong>',
+          },
+          {
+            description: 'Their circumstances may have changed during the coronavirus pandemic. Answer all questions based on their situation as it is now.',
+          },
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet:41]:
```suggestion
          {
            title: 'Coronavirus (COVID-19)',
            description: 'Their circumstances may have changed during the coronavirus pandemic. Answer all questions based on their situation as it is now.',
          },
```

File: source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet [hunk 9]
```diff
@@ -1,12 +1,30 @@
...
+  }
+  else 'How do you usually travel to work?'
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then [
+    'Answer for the longest part, <em>by distance</em>, of their current journey to work',
+    'If the <strong>coronavirus</strong> pandemic has affected how they travel to work, select the answer that best describes their <strong>current circumstances</strong>.',
+  ] else [
+    'Answer for the longest part, <em>by distance</em>, of your current journey to work',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet:19]: I can see these were wrong prior to the changes, but these should end with a full stop.
```suggestion
    'Answer for the longest part, <em>by distance</em>, of their current journey to work.',
    'If the <strong>coronavirus</strong> pandemic has affected how they travel to work, select the answer that best describes their <strong>current circumstances</strong>.',
  ] else [
    'Answer for the longest part, <em>by distance</em>, of your current journey to work.',
```

File: source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet [hunk 10]
```diff
@@ -1,12 +1,30 @@
...
+    'If the <strong>coronavirus</strong> pandemic has affected how you travel to work, select the answer that best describes your <strong>current circumstances</strong>.',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet:20]: This applies to all text in question descriptions - why is there a mix of `strong` and `em` tags?

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet [hunk 11]
```diff
@@ -71,19 +78,19 @@ local additionalAnswerOption = [
...
       when: [rules.isNotProxy, rules.isNotFirstPersonInList(listName)],
     },
     {
-      question: question(proxyTitle, proxyDescription),
+      question: question(isProxy=true),
       when: [rules.isProxy, rules.isFirstPersonInList(listName)],
     },
     {
-      question: question(proxyTitle, proxyDescription, additionalAnswerOption),
+      question: question(additionalAnswerOption, isProxy=true),
```
[the remark this answers is not in the document]
Comment (berroar) [reply source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet:93]: It won't build the schema if I swap the order, i get this error `RUNTIME ERROR: internal error: got positional param after named at index 1`

File: source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet (hunk 10 above)
Comment (berroar) [reply source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet:20]: I've tried just to use strong throughout bust missed these ones

File: source/jsonnet/england-wales/individual/blocks/employment/hours_worked.jsonnet (hunk 6 above)
Comment (berroar) [reply source/jsonnet/england-wales/individual/blocks/employment/hours_worked.jsonnet]: I've added full stops to all the ones you mentioned, I'll ask Donna to take a look today before I merge

File: source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet (hunk 11 above)
[the remark this answers is not in the document]
Comment (ajmaddaford) [reply source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet]: The order would need to be changed in the `question` function, and called with them both as positional arguments:
```
question(isProxy=true, additionalAnswerOptions=additionalAnswerOption)
```

The other option is to make it consistent with `isProxy` and do something like:
```
question(isProxy=true, isFirstPerson=true)
```

File: source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet (hunk 10 above)
Comment (ajmaddaford) [reply source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet:20]: I think we should check which one should be used, as there is significance to using `em` over `strong`.

File: source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet (hunk 11 above)
[the remark this answers is not in the document]
Comment (berroar) [reply source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet]: I've gone with the second option and made it consistent with isProxy now 👍

File: source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet [hunk 12]
```diff
@@ -3,18 +3,49 @@ local rules = import 'rules.libsonnet';
...
+  if isFirstPerson then [{
+    label: {
+      text: 'Same as {first_person_possessive} address one year ago',
+      placeholders: [
+        placeholders.firstPersonNamePossessiveForList(listName),
+      ],
+    },
+    value: 'Same as {first_person_possessive} address one year ago',
+  }]
+  else []
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet:26]: This is the wrong way round. If it is the first person, they shouldn't get the option to say the address is the same as the first person.

File: source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet (hunk 8 above)
Comment (ajmaddaford) [reply source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet:39]: I think this still needs to be updated.

Comment (berroar) [reply source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet]: Sorry missed the second one, fixed now

**Review by ajmaddaford — APPROVED** [review]

**Comment by berroar** [comment]
All content checked by Donna

**Pull request merged**